data/transport missions.txt

```diff
@@ -1951,3 +1951,383 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Lost Boy 1"
+	minor
+	name `Rescue Tod`
+	description `Travel to <destination> in search for Tod, the son of a single mother from <origin>.`
+	source "Moonshake"
+	destination "Deadman's Cove"
+	to offer
+		"combat rating" > 100
+	
+	on offer
+		conversation
+			`Wandering around the spaceport to see what it has to offer, you make your way into one of the dimly lit restaurants to avoid the crowded streets and metallic tasting air outside. The restaurant is mostly empty. One of its only inhabitants is a dirty looking woman crying with her head down in one of the booths.`
```

Comment:
`metallic-tasting`

or maybe `musty`, `dense`, `stuffy`

Reply:
Describing the air as tasting metallic is pulled from the spaceport description.

Reply:
Needs the hyphen then

Reply:
Idk about changing that. While it technically is correct, combining words by hyphenating them like that is, to my understanding, on a down trend. The no-good case that I fixed is a different situation though. "No good" means without good, but "no-good" means having no value or being worthless; the meaning in the situation is changed. In this situation, hyphenating "metallic tasting" isn't necessary to get the proper meaning across.

Reply:
Doesn't mean we have to help accelerate the demise of grammar :grinning: 

:older_man: 
:sob:

Reply:
Pwease give it a dash? :3 kthx Derp <3

Reply:
In this case, a hyphen is required because that's a compound adjective; the hyphen makes it clear that the air is metallic-tasting, so no one will think that the tasting air (like a tasting menu?) is metallic. (On the other hand, you'd have to check if adding the hyphen messes up the word wrap and justification.)